packages/nextjs/app/builders/0xC52E3E454a735d4A74F3dd6Ea281F7AebDFAad83/page.tsx

```diff
@@ -0,0 +1,98 @@
+import Image from "next/image";
+import ProfilePic from "./pfp.png";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const PersonalPage: NextPage = () => {
+  return (
+    <div className="min-h-screen bg-[#F9F9F9] text-[#333333] flex flex-col items-center justify-center">
+      {/* Profile Picture */}
+      <div className="mb-6">
+        <Image
+          src={ProfilePic}
+          alt="Profile"
+          width={128}
+          height={128}
+          className="rounded-full border-4 border-[#EAEAEA] shadow-lg"
+        />
+      </div>
+
+      {/* Name */}
+      <h1 className="text-4xl font-bold text-[#1A1A1A] mb-4">Cherry 🍒</h1>
+
+      {/* Description */}
+      <p className="text-lg text-[#4F4F4F] max-w-md text-center mb-8">
+        I am currently pursuing a Bachelor&apos;s degree in Computer Science and I aspire to become a blockchain
+        developer and build cool things. Nice to meet you! 🌟
+      </p>
+
+      {/* Social Media Links */}
+      <div className="flex space-x-6 mb-8">
+        <a
```

Comment:
For a cleaner code base you might want to create an array with your social links and use a map function to render it.